README.md

```diff
@@ -15,7 +16,7 @@ Presentation on The Importance of Software Architecture -- see [pages 3-7](http:
 
 ### What Is Software Architecture Design?
 
-- ##### Architecture is a *general* term for the overall structure of a software system. Fundamentally, it's about managing complexity.
+- ##### Architecture is a _general_ term for the overall structure of a software system. Fundamentally, it's about managing complexity.
```

Comment:
Can we make this an `h4`? Since the closest parent is an `h3`—we shouldn't skip heading levels.

```suggestion
- #### Architecture is a _general_ term for the overall structure of a software system. Fundamentally, it's about managing complexity.
```

Reply:
There are a few other places where we jump from `h2` to `h5` etc as well that we should fix.